package.json

```diff
@@ -7,7 +7,9 @@
     "lint:js": "eslint .",
     "lint:css": "stylelint blocks/**/*.css styles/*.css",
     "lint": "npm run lint:js && npm run lint:css",
-    "semantic-release": "semantic-release --debug"
+    "semantic-release": "semantic-release --debug",
+    "start": "aem up --print-index",
+    "postinstall": "rm -rf scripts/dropins && cp -R node_modules/@dropins scripts/dropins && find scripts/dropins -type f -name \"package.json\" -exec rm {} \\;"
```

Comment:
@herzog31 how do you feel about this approach long term? Maybe we should instead have a script that can be called by an integrator/merchant dev when intended, rather than on post-install. Would avoid incidental upgrades.

I wish we could avoid having to copy dependency files at all.

Reply:
I think this is fine for now. Incidental upgrades should not happen, since the version of the node dependency should be locked.